.gitignore

```diff
@@ -7,6 +7,7 @@ node_modules/
 # IDEs
 .vscode
 .idea
+*.iml
```

Comment:
whats iml for? :suspect:

Reply:
Those are IntelliJ project files on top of the `.idea` directory. They are not always created, haven't found the pattern when IntelliJ creates them. We have taken the habit of always adding those to the `.gitignore` alongside the `.idea` directory.